source/_integrations/ezviz.markdown

```diff
@@ -15,6 +15,13 @@ It also exposes an RTSP stream, by using the local camera IPs (so the device hos
 
 As there is no official documentation for the API, the component retrieves data from the API used in the Ezviz mobile app, [hosted here](https://apiieu.ezvizlife.com).
 
+
+```yaml
```

Comment:
Make it a warning -> https://developers.home-assistant.io/docs/documenting/create-page#html